src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest.kt

```diff
@@ -910,59 +906,440 @@ class OffenderResourceIntTest : ResourceTest() {
       }
 
       val resp = webTestClient.post().uri("/api/offenders/A1176RS/case-notes")
-        .headers(setAuthorisation("ITAG_USER", listOf("")))
+        .headers(setAuthorisation("ITAG_USER", listOf()))
         .header("Content-Type", APPLICATION_JSON_VALUE)
-        .body(
-          BodyInserters.fromValue(newCaseNote),
-        )
+        .bodyValue(newCaseNote)
         .exchange()
         .expectStatus().isOk
 
       val cn = resp.returnResult(CaseNote::class.java).responseBody.blockFirst()!!
       return cn.caseNoteId
     }
+  }
 
-    private fun removeCaseNoteCreated(caseNoteId: Long) {
-      val ocn = offenderCaseNoteRepository.findById(caseNoteId).get()
-      offenderCaseNoteRepository.delete(ocn)
+  @Nested
+  @DisplayName("POST /api/offenders/{offenderNo}/case-notes")
```

Comment:
The majority of these tests were pulled in from the now removed create case note by bookingId (with minor change to use offenderNo instead of bookingId)
A few additional ones were also added to test authorisation